nodeJS/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "nodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js"
```

Comment:
For development you could use **nodemon**

Reply:
I've added. Thanks